idd/Energy+.idd.in

```diff
@@ -67194,6 +67249,33 @@ AirLoopHVAC:ReturnPath,
        \type object-list
        \object-list ReturnPathComponentNames
 
+AirLoopHVAC:ExhaustSystem,
+       \memo Defines a general exhaust systems with a central exhaust fan drawing from one or more
+       \memo ZoneHVAC:ExhaustControl outlet nodes.
+  A1 , \field Name
+       \required-field
+       \note Name of the exhaust system
+  A2 , \field Availability Schedule Name
```

Comment:
At one point in our (many) discussions, we discussed eliminating this field, because the fan has it's own schedule.

Reply:
This field now removed.